src/main/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrLetterPageTask.java

```diff
@@ -0,0 +1,52 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mserver.crawler.wdr.WdrConstants;
+import de.mediathekview.mserver.crawler.wdr.WdrTopicUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrLetterPageDeserializer;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrLetterPageUrlDeserializer;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+public class WdrLetterPageTask implements Callable<ConcurrentLinkedQueue<WdrTopicUrlDTO>>  {
+
+  private static final Logger LOG = LogManager.getLogger(WdrLetterPageTask.class);
+  
+  private final WdrLetterPageDeserializer deserializer = new WdrLetterPageDeserializer();
+  
+  @Override
+  public ConcurrentLinkedQueue<WdrTopicUrlDTO> call() throws Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)